src/run_environment/reference_genome.ml

```diff
@@ -175,6 +175,9 @@ module Default = struct
     let dbsnp_mm10_indels_url =
       "ftp://ftp-mouse.sanger.ac.uk/REL-1303-SNPs_Indels-GRCm38/\
        mgp.v3.indels.rsIDdbSNPv137.vcf.gz" in
+    let gene_annotations_gtf =
+      "ftp://ftp.ensembl.org/pub/release-79/gtf/mus_musculus/\
```

Comment:
Just a small note: an Ensembl release doesn't necessarily correspond to a
different minor patch of a genome. They release on a regular schedule and
sometimes it's just an update to the genome metadata (e.g. transcript
intervals, biotype of a gene, &c) while still using the exact same genome
sequence as the previous release.

On Tue, Apr 19, 2016 at 1:42 PM, Isaac Hodes <notifications@github.com>
wrote:

> In src/run_environment/reference_genome.ml
> <https://github.com/hammerlab/biokepi/pull/244#discussion_r60276449>:
>
> > @@ -175,6 +175,9 @@ module Default = struct
> >      let dbsnp_mm10_indels_url =
> >        "ftp://ftp-mouse.sanger.ac.uk/REL-1303-SNPs_Indels-GRCm38/\
> >         mgp.v3.indels.rsIDdbSNPv137.vcf.gz" in
> > +    let gene_annotations_gtf =
> > +      "ftp://ftp.ensembl.org/pub/release-79/gtf/mus_musculus/\
>
> #245 <https://github.com/hammerlab/biokepi/issues/245>
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/biokepi/pull/244/files/6669925b070543cdabab80c4972bb4c428b6b8a5#r60276449>
>